mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -280,10 +279,15 @@ var schemaNodePool = map[string]*schema.Schema{
 					ForceNew:     true,
 					Description:  `Whether to create a new range for pod IPs in this node pool. Defaults are provided for pod_range and pod_ipv4_cidr_block if they are not specified.`,
 				},
-
+				"enable_private_nodes": {
```

Comment:
Looks like we also add `enable_private_nodes` to `google_container_node_pool` resource. Would you mind adding this to the changelog. Plus, do we aslo need to add this to the doc at mmv1/third_party/terraform/website/docs/r/container_node_pool.html.markdown?

Reply:
Sure, I have added a note about `enable_private_nodes` to the changelog. For the markdown file, it has been already added.